src/utils/feature-flags.ts

```diff
@@ -212,6 +212,16 @@ export const FF_LSDV_4701 = 'fflag_feat_front_lsdv_4701_audio_default_decoder_ff
  */
 export const FF_LSDV_4659 = 'fflag_feat_front_lsdv_4659_skipduplicates_060323_short';
 
+Object.assign(window, {
+  APP_SETTINGS: {
+    ...(window.APP_SETTINGS ?? {}),
+    feature_flags: {
+      ...(window.APP_SETTINGS?.feature_flags ?? {}),
+      ...(window.__FEATURE_FLAGS__ ?? {}),
```

Comment:
Why can't we just change `APP_SETTINGS` without additional global var?

Reply:
true. probably don't need it at all

Reply:
turned out that without it feature flags that are set before page is loaded are overriden somehow. I will keep it in place for safety